src/promptflow/tests/sdk_cli_test/e2etests/test_executable.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import mock
+import pytest
+
+from ..recording_utilities import RecordStorage
+from .test_cli import run_pf_command
+
+FLOWS_DIR = "./tests/test_configs/flows"
+RUNS_DIR = "./tests/test_configs/runs"
+CONNECTIONS_DIR = "./tests/test_configs/connections"
+DATAS_DIR = "./tests/test_configs/datas"
+
+
+@pytest.mark.skipif(RecordStorage.is_replaying_mode(), reason="Missing executable deps in replay mode")
+@pytest.mark.usefixtures("use_secrets_config_file", "setup_local_connection", "install_custom_tool_pkg")
+@pytest.mark.cli_test
+@pytest.mark.e2etest
+class TestExecutable:
```

Comment:
is this test moved from other file? seems not related to replay test

Reply:
Yes, move out of test_cli, just a small refactor.

This test is skipped because it requires promptflow[executable]